borg/archive.py

```diff
@@ -777,7 +804,7 @@ def missing_chunk_detector(chunk_id):
             num_archives = 1
             end = 1
         for i, (name, info) in enumerate(archive_items[:end]):
-            self.report_progress('Analyzing archive {} ({}/{})'.format(name, num_archives - i, num_archives))
+            logger.info('Analyzing archive {} ({}/{})'.format(name, num_archives - i, num_archives))
```

Comment:
i must say i am a little wary of all those wrappers to print and logging calls. that was a quick fix to restore previous behaviour but okay, i'll revert.